components/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import useTheme from "../use-theme";
+
+interface Props {
+    width?: number
+    squared?: boolean
+    rounded?: boolean
+    component?: keyof JSX.IntrinsicElements
+    className?: string
+    show?: boolean
+    minHeight?: number
+    animate?: boolean
+    height?: number
+}
+
+const defaultProps = {
+    squared: false,
+    rounded: false,
+    component: 'span' as keyof JSX.IntrinsicElements,
+    className: '',
+    show: false,
+    minHeight: 24,
+    animate: true,
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type SkeletonProps = Props & NativeAttrs
+
+const Skeleton: React.FC<React.PropsWithChildren<SkeletonProps>> = ({
+                                                                        component,
+                                                                        children,
+                                                                        width,
+                                                                        squared,
+                                                                        rounded,
+                                                                        show,
+                                                                        minHeight,
+                                                                        className,
+                                                                        animate,
+                                                                        height,
+                                                                        ...props
+                                                                    }: React.PropsWithChildren<SkeletonProps> & typeof defaultProps) => {
```

Comment:
We also need to introduce useScale to make the component compatible with scaling, ref: [badge.tsx#L44](https://github.com/geist-org/geist-ui/blob/master/components/badge/badge.tsx#L44)

Reply:
I added scaling to width and height